### DISCUSSION TITLE
Bycicle and Public Transport routing

### DISCUSSION THREAD

**Discussion by cutec-chris** [osmandapp/OsmAnd#20603] [General]
Is there an possibility to create an Profile for instead of walk+Public Transport -> have bike + Public Transport ?
Basically the difference could be Speed of the "walking" part, better would be to have real bike routing.

**Comment by DmitryAlexei** [comment]
You can create your own routing for a new profile
For more information see https://osmand.net/docs/user/navigation/routing/osmand-routing#customize-offline-routing
